drivers/SmartThings/matter-switch/profiles/light-color-level-1800-6500K.yml

```diff
@@ -0,0 +1,22 @@
+name: light-color-level-1800-6500K
```

Comment:
This should be `light-color-level-1800K-6500K` (notice the extra K after 1800) to match naming scheme of other similar fingerprints. Please change the `name:` and the filename, then update the `deviceProfileName` in the fingerprints to match ``light-color-level-1800K-6500K`